### ISSUE TITLE
Error: Cannot find module 'bluebird' | v0.14.0

### ISSUE THREAD

**Issue by bensooraj** [moleculerjs/moleculer#674]
## Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository

## Current Behavior

Running the following example code:
```js
const { ServiceBroker } = require("moleculer");

// Create a ServiceBroker
const broker = new ServiceBroker();

// Define a service
broker.createService({
    name: "math",
    actions: {
        add(ctx) {
            return Number(ctx.params.a) + Number(ctx.params.b);
        }
    }
});

// Start the broker
broker.start()
    // Call the service
    .then(() => broker.call("math.add", { a: 5, b: 3 }))
    // Print the response
    .then(res => console.log("5 + 3 =", res))
    .catch(err => console.error(`Error occured! ${err.message}`));
```

Throws an error: *`Error: Cannot find module 'bluebird'`*

## Expected Behavior
It should print the following:
```
"5 + 3 = 8"
```

### Steps to Reproduce

Additionally, following is what the `package.json` file looks like:
```json
{
  "name": "service-1",
  "version": "1.0.0",
  "description": "",
  "main": "main.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "express": "^4.17.1",
    "moleculer": "^0.14.0",
    "moleculer-web": "^0.9.0"
  }
}
```

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Moleculer version: 0.14.0
* NodeJS version: v12.1.0
* Operating System: macOS Sierra

### Failure Logs
```sh
$ node main.js 
internal/modules/cjs/loader.js:613
    throw err;
    ^

Error: Cannot find module 'bluebird'
Require stack:
- /User/Desktop/NodeJS/m2/node_modules/moleculer/src/transporters/amqp10.js
- /Users/Desktop/NodeJS/m2/node_modules/moleculer/src/transporters/index.js
- /Users/Desktop/NodeJS/m2/node_modules/moleculer/src/service-broker.js
- /Users/Desktop/NodeJS/m2/node_modules/moleculer/index.js
- /Users/Desktop/NodeJS/m2/main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.Module._load (internal/modules/cjs/loader.js:526:27)
    at Module.require (internal/modules/cjs/loader.js:666:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/Bensooraj/Desktop/NodeJS/m2/node_modules/moleculer/src/transporters/amqp10.js:10:17)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Module.require (internal/modules/cjs/loader.js:666:19)
```

**Comment by intech** [comment]
I confirm the problem.
@bensooraj temp fix in package.json
```js
  "dependencies": {
    "bluebird": "latest",
    "moleculer": "0.14.0"
  }
```

**Comment by icebob** [comment]
Thanks @bensooraj 
Interesting because Travis was green.

**Comment by icebob** [comment]
Fixed in 0.14.1

**Comment by bensooraj** [comment]
Thanks!
